src/utils/buffer.ts

```diff
@@ -0,0 +1,12 @@
+import assert from 'assert';
+import buffer from 'buffer';
+
+const isHexa = (value: string): boolean => {
+  // test if value is string?
+  return /^[0-9a-fA-F]*$/.test(value);
+};
+
+export const hexToBuffer = (value: string): Buffer => {
+  assert(isHexa(value));
```

Comment:
Does it work in the browser? I'm wondering whether it will work in hathor-wallet-desktop because it runs through Electron. Maybe we should use just raise an exception to avoid compatibility issues.